readme.md

```diff
@@ -18,7 +18,7 @@
 </p>
 
 <p align="center">
-  <a href="#packages">Packages</a> • <a href="#components">Components</a> • <a href="#providers">Providers</a> • <a href="#getting-started">Getting Started</a> • <a href="#running-the-samples">Running the Samples</a> • <a href="#contribute">Contribute</a> • <a href="#feedback-and-requests">Feedback & Requests</a> <br>• <a href="#license">License</a> • <a href="#code-of-conduct">Code of Conduct</a>
```

Comment:
Do we want to have a "Running the sample" section instead?

Reply:
We should have it. So that it can also mention the location of the other samples.

Reply:
I like this suggestion. I'll add it back and reference the Samples repo!